### ISSUE TITLE
Support for year-end charts

### ISSUE THREAD

**Issue by charlesverdad** [guoguo12/billboard-charts#33]
Year-end charts does not seem to be supported at the moment.

It would be great if the library can use `https://www.billboard.com/charts/year-end/YEAR/CHARTNAME` in addition to  `https://www.billboard.com/charts/CHARTNAME`

**Comment by ProfoundWanderer** [comment]
@guoguo12 Can I be contributor so I can make a branch for this issue?

**Comment by jameswenzel** [comment]
@ProfoundWanderer I see you've already forked the repo, you can just make a branch off that and submit a pull request to the main repo.

**Comment by ProfoundWanderer** [comment]
@jameswenzel Ok cool I didn't know the best way to do it

**Comment by nelsonlage** [comment]
I have just stumbled upon this .api and you did a great job! I was wondering if you guys have solved the year-end chart issue. I'm pretty new to GitHub and had a look in the branches, but it wasn't clear to me. Thanks for taking the time to do this.

**Comment by guoguo12** [comment]
I don't think so. You could try running the code in @ProfoundWanderer's branch though.

**Comment by ProfoundWanderer** [comment]
> I have just stumbled upon this .api and you did a great job! I was wondering if you guys have solved the year-end chart issue. I'm pretty new to GitHub and had a look in the branches, but it wasn't clear to me. Thanks for taking the time to do this.

I haven't looked at my branch in a long time but if you have any questions about it feel free to ask.

**Comment by alvst** [comment]
Hi @ProfoundWanderer, I was taking a look at your code trying to complete it as I was also hoping to put together something that used yearend charts. I've been looking at it and it trying to figure out why every time I run it I get a 403 Error. I tried a diff check to see if I could figure out the offending code but can't seem to find it. 
Anyone else seeing this error? Maybe something changed between then and now that has already been merged into the master branch?
